tests/unit/test_runner_manager.py

```diff
@@ -174,10 +174,27 @@ def test_update_runner_bin(runner_manager: RunnerManager):
     """
 
     class MockRequestLibResponse:
-        def __init__(self, *arg, **kargs):
+        """A mock requests library response."""
+
+        def __init__(self, *args, **kwargs):
+            """Initialize successful requests library response.
+
+            Args:
+                args: Placeholder for positional arguments.
+                kwargs: Placeholder for keyword arguments.
+            """
             self.status_code = 200
 
-        def iter_content(self, *arg, **kargs):
+        def iter_content(self, *args, **kwargs):
+            """Mock content iterator reteruning an iterator over a single test runner binary.
```

Comment:
```suggestion
            """Mock content iterator returning an iterator over a single test runner binary.
```

Reply:
Thanks for catching all the errors/typos @cbartz !